assets/src/design-system/components/toggle/stories/index.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import styled from 'styled-components';
+import { action } from '@storybook/addon-actions';
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import { Toggle } from '..';
+import { Text } from '../..';
+import { DarkThemeProvider } from '../../../storybookUtils';
+import { Headline } from '../../..';
+
+export default {
+  title: 'DesignSystem/Components/Toggle',
+  component: Toggle,
+};
+
+const Container = styled.div`
+  display: grid;
+  row-gap: 20px;
+  padding: 20px 40px;
+  background-color: ${({ theme }) => theme.colors.bg.primary};
+  border: 1px solid ${({ theme }) => theme.colors.standard.black};
+  max-width: 400px;
+`;
+
+const Row = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-column: 1 / -1;
+  label {
+    display: flex;
+    align-items: center;
+  }
+`;
+
+export const _default = () => {
+  const [inputState, setInputState] = useState({
+    oneLight: false,
+    twoLight: true,
+    threeLight: false,
+    fourLight: true,
+    oneDark: false,
+    twoDark: true,
+    threeDark: false,
+    fourDark: true,
+  });
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.checked;
+
+    action(event.target.name)(event);
+    setInputState((prevState) => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
+
+  return (
+    <>
+      <Headline as="h1">{'Toggle'}</Headline>
+      <br />
+      <Container>
+        <Row>
+          <Text>{'Normal'}</Text>
+          <Text>{'Normal'}</Text>
+          <Text>{'Disabled'}</Text>
+          <Text>{'Disabled'}</Text>
+        </Row>
+        <Row>
+          <Toggle
+            id="one-light"
+            name="oneLight"
+            checked={inputState.oneLight}
+            onChange={handleChange}
+            label={text('Toggle 1 Label')}
```

Comment:
nit: i know that the props are getting spread but for the sake of a followable best practice demo can you add in ariaLabel here so that accessibility screening picks up the label? 

![Screen Shot 2021-02-03 at 9 23 10 AM](https://user-images.githubusercontent.com/10720454/106776599-75114a80-6601-11eb-9361-92b1f8d341fd.png)